webapi/types.go

```diff
@@ -71,3 +71,16 @@ type ticketStatusResponse struct {
 	VoteChoices     map[string]string `json:"votechoices"`
 	Request         []byte            `json:"request"`
 }
+
+type SetAltSigRequest struct {
+	Timestamp     int64  `json:"timestamp" binding:"required"`
+	TicketHash    string `json:"tickethash" binding:"required"`
+	TicketHex     string `json:"tickethex" binding:"required"`
+	ParentHex     string `json:"parenthex" binding:"required"`
+	AltPubAddress string `json:"altpubaddress"`
```

Comment:
Should this be `AltSigAddress`? And shouldn't it be required?

Reply:
Ah, yes.

Reply:
0175871
Also changed in the decrediton pr so it works.